packages/commands/diff/src/static/index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
```

Comment:
@kamilkisiela Need your help. How to include (copy) this file into build?

Reply:
We would have to adjust [`bob-the-bundler`](https://github.com/kamilkisiela/bob) but I'll do it after merging this PR.